cmd/sqlflowserver/main_test.go

```diff
@@ -677,11 +677,10 @@ func CaseTrainBoostedTreesEstimatorAndExplain(t *testing.T) {
 	SELECT * FROM iris.train WHERE class!=2
 	TO TRAIN BoostedTreesClassifier
 	WITH
-		model.n_batches_per_layer=8,
+		model.n_batches_per_layer=10,
 		model.n_trees=50,
-		model.n_classes=2,
```

Comment:
That is superfluous because the `BoostedTreesClassifier` is only capable of binary classification.
Of course, removing this has nothing to do with performance stability:-)

Reply:
Maybe we need to `assert model.n_cleasses == 2` instead of moving this argument.

Reply:
I think it's ok without an assertion, all `classifier`s in `tf.estimator` defaults to binary classification.